### ISSUE TITLE
Doesn't work on RPI4 with MPEG2 streams

### ISSUE THREAD

**Issue by nhathaway** [ogra1/kodi-pi-standalone#4]
I know that the hardware decoder does not support MPEG2, and that on RPI4 you cannot but a licence for MPEG2. MPEG2 decoding has to be done in software. I understand that Kodi is supposed to fallback to software decoding, but it's not doing that. You get sound but no video.

Is there some setting somewhere? I couldn't find one

**Comment by ogra1** [comment]
no, there is no setting there is likely something needed in the internal kodi ffmpeg build to support the MPEG2 fallback, i sadly did not have the time to research this in depth yet (it works fine on a pi3 that has the license installed btw)
